assets/src/edit-story/components/form/select/select.js

```diff
@@ -18,19 +18,19 @@
  * External dependencies
  */
 import styled, { css } from 'styled-components';
+import { DropDown as DropDownSelect } from '@web-stories-wp/design-system';
 
 /**
  * Internal dependencies
  */
-import { DropDown } from '../../../../design-system';
 import { focusStyle } from '../../panels/shared';
 
 const Container = styled.div`
   position: relative;
   width: 100%;
 `;
 
-const StyledDropDown = styled(DropDown)`
+const StyledDropDown = styled(DropDownSelect)`
```

Comment:
@miina did you see this one, is this ok?

Reply:
Why is `DropDown` imported as `DropDownSelect` here? Why the renaming? 🤔

Reply:
Thanks for tagging. It doesn't make a difference since it's just renaming it but not sure why -- @spacedmonkey Any specific reason for renaming `DropDown` to `DropDownSelect` here? Let's change it back, if not.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/8087/commits/eecb244e0bc64ee01035f6d538a6e57c6e0cc59a